src/autopas/tuning/tuningStrategy/ruleBasedTuning/RuleVM.h

```diff
@@ -235,8 +235,9 @@ class RuleVM {
         break;
       }
       case DIV: {
-        auto res =
-            computeBinary(_stack.at(_stackPointer - 1), _stack.at(_stackPointer), [](auto l, auto r) { return l / r; });
+        auto res = computeBinary(_stack.at(_stackPointer - 1), _stack.at(_stackPointer), [](auto l, auto r) {
+          return r == 0 ? std::numeric_limits<decltype(r)>::max() : l / r;
+        });
```

Comment:
@HobbyProgrammer, what's your opinion on this? I know it's mathematically not correct but in the terms of rules yields the intuitively correct relation results. Also I didn't want to use `inf` because it is only defined for floating point numbers, thus having different behaviors for `int` and `float`.
The problem is encountered if any rule or `define` in the rule file for example divides by the number of particles.